src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./Introduce.css";
+
+const image = () => {
```

Comment:
```suggestion
const Introduce = () => {
```

컴포넌트의 이름은 컴포넌트 파일 이름과 동일하게 구현해주세요!